draft-ietf-jsonpath-base.md

```diff
@@ -246,7 +246,11 @@ Unicode Scalar Value:
   E000 to 10FFFF. JSON values of type string are sequences of Unicode scalar values.
 
 Singular Path:
-: A JSONPath expression built from selectors which each select at most one node.
+: A JSONPath expression built from appenders which each select at most one node.
+
+Selector:
+: A single item within a bracketed (`[]`) child appender that matches the values which
+  are to be selected.
```

Comment:
* This definition positions a selector as something which _matches_, leaving _selecting_ to the child appender. That division is not consistent with the rest of the document.

* Selectors also appear with descendant appenders.

* "the values" is a bit vague. I think we need to say that a selector selects zero or more children of a given value.

Reply:
This definition could point us to a way out of the weakness in the "appender"/"selector" terminology, Glyn was mentioning.

If we re-rename "selector" to what it has been before and call "selector items" (or "items" in short) those items inside the brackets, above definition reads:

```
Item:
: A single item within a bracketed (`[]`) child selector that matches the values which
  are to be selected.
```
Use of `selector`/ `selector-item` would also relax the need of strict differentiating "selecting" / "matching" terminology (which is useful though).